pkg/cmd/output.go

```diff
@@ -108,6 +108,14 @@ func makeAllocationTable(allocationType string, allocations map[string]kubecost.
 		AlignFooter: text.AlignRight,
 	})
 
+	if opts.showEfficiency {
+		columnConfigs = append(columnConfigs, table.ColumnConfig{
+			Name:        "Total Efficiency",
```

Comment:
Should this be "Cost" or "Spend" efficiency? I would want to highlight that this is not a straight utilization calculation.

Reply:
I think so. Changed to "Cost Efficiency."